### PR TITLE
feat: utils - re-export @polkadot/util as dotUtils

### LINKED ISSUE #29: Create helper functions for formatting Big Numbers in BN.js
## Feature Request

* Create helper functions that take a BN.js, string, or number as a NUMBER input and format it in human readable ways. Each api registry has chain decimal information, which should be used to properly format decimals, etc. 

### Suggestion

`useBalance()?.freeBalance` will return something like `1096562700458522` for Contracts on Rococo.

* We should be able to easily display that as `1,196.56 ROC`, `1196.5627`, `1096.562700458522`, etc. We can control how many significant numbers are shown, show the currency, use commas, etc. 
* Add helpers for decimal math without floating point numbers. e.g. Chain A uses 12 decimals and Chain B uses 18 decimals. If we want to calculate an exchange rate between the two we want to do this easily.

### Motivation

Formatting BN is tedious, and very common.

### Use Cases

Displaying a user's balance with only two decimals. e.g. `1,096.56 ROC`

### PR DESCRIPTION
**Pull request by nicolad**
Resolves #29

- [ ] There is an associated issue (**required**)
- [ ] The change is described in detail
- [x] There are new or updated tests validating the change (if applicable)

image

### CODE REVIEW

File: core/providers/temp.test.ts [hunk 1]
```diff
@@ -1,5 +0,0 @@
-import { assert } from 'https://deno.land/std@0.174.0/testing/asserts.ts';
```
Comment (nicolad) [inline core/providers/temp.test.ts:1]: @DoubleOTheven hope the removal is fine here

File: core/utils/mod.ts [hunk 2]
```diff
@@ -3,3 +3,4 @@ export * from './contracts/mod.ts';
 export * from './getExpiredItem.ts';
 export * from './parseUnits.ts';
 export * from './pseudoRandomId.ts';
+export * from './units/mod.ts';
```
Comment (nicolad) [inline core/utils/mod.ts:6]: I saw `mod.ts` approach in a different folder and followed up that.

File: core/providers/contractEvents/reducer.ts [hunk 3]
```diff
@@ -14,7 +14,10 @@ interface RemoveContractEvent {
 
 type Action = AddContractEvent | RemoveContractEvent;
 
-export function contractEventReducer(state: ContractEvents, action: Action): ContractEvents {
+export function contractEventReducer(
```
Comment (DoubleOTheven) [inline core/providers/contractEvents/reducer.ts:17]: It looks like there is a lot of formatting. Does your IDE have a different config for max character length, etc?

Comment (nicolad) [reply core/providers/contractEvents/reducer.ts:17]: it was just `deno fmt`, should I revert this change?

Comment (nicolad) [reply core/providers/contractEvents/reducer.ts]: reverted the change, if needed or if u want can create a separate PR with `deno fmt`

**Comment by DoubleOTheven** [comment]
I just exported everything using the `deno -> dnt` way. 

[This](https://github.com/paritytech/useink/blob/cc8bba6c7b51cd1244f29cf6b9624aaddf8441db/_tasks/dnt.ts#L22) is how you set up a new import path. e.g. `import * as U from 'useink/utils'`

And I [export here](https://github.com/paritytech/useink/blob/main/utils/mod.ts)

**Pull request closed without merging**